src/App.scss

```diff
@@ -8,6 +8,13 @@ body,
   width: 100%;
   padding: 0;
   margin: 0;
+  height: 100vh;
```

Comment:
Any specific reason why the page has a fixed height?

Reply:
Ohhh that's a typo it can be height 100% too. But I need a dimension regarding height there so that sticky will know its parent height.

And yes i can remove that overflow. But just specified there to avoid browser automatic decision.